semantic_conventions/trace/feature-flag.yaml

```
@@ -0,0 +1,35 @@
+groups:
+  - id: feature_flag
+    prefix: feature_flag
```

```suggestion
    prefix: feature_flag
    type: span
```

see https://github.com/open-telemetry/opentelemetry-specification/runs/8159209071